R/httr2-translate.R

```diff
@@ -0,0 +1,189 @@
+#' Translate httr2 request to curl command
+#'
+#' Convert an httr2 request object to equivalent curl command line syntax.
+#' This is useful for debugging, sharing requests, or converting to other tools.
+#'
+#' @param .req An httr2 request object created with [request()].
+#' @return A character string containing the curl command.
```

Comment:
Can you please add a `@seealso` link to `curl_translate()` and one from `curl_translate()` back here?